internal/orchestrator/repo/repo.go

```diff
@@ -311,17 +311,25 @@ func (r *RepoOrchestrator) Restore(ctx context.Context, snapshotId string, snaps
 	opts = append(opts, restic.WithFlags("--target", target))
 
 	if snapshotPath != "" {
+		normalizedPath := strings.ReplaceAll(snapshotPath, "\\", "/")
+
+		dir := path.Dir(normalizedPath)
+		base := path.Base(normalizedPath)
+		
 		if runtime.GOOS == "windows" {
-			opts = append(opts, restic.WithFlags("--include", snapshotPath))
+		    if dir != "" {
```

Comment:
is this branch now the same as the branch for linux? It may be fine to combine them.